app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/FirebaseHelper.java

```diff
@@ -97,7 +214,7 @@ public void addUser(User user, final FirebaseCallback<DocumentReference> callbac
      * add a mood event to the database
      *
      * @param moodEvent MoodEvent to be added
-     * @param photo photo to be attached
+     * @param photo     photo to be attached
```

Comment:
Lots of added white space in this file. is that intentional?

Reply:
Yes, it was automated formatting.